### PR TITLE
Change to Log Font and Formatting

### PR DESCRIPTION
**Pull request by kunalgosar**
Different font on log window (Courier) and colored/bolded timestamps. Feedback welcome!

New log window looks like:
final 

In response to: https://highfidelity.fogbugz.com/f/cases/3322/Font-in-log-window-is-hard-to-read

### CODE REVIEW

**Comment by kunalgosar** [comment]
@OmegaHeron @Nex-Pro Thank you for your feedback on the other PR. I was having git issues and so have opened a new PR. 

Wrt @Nex-Pro's reported bug about the timestamps unbolding when searching: that should be resolved in this PR.

I agree that there should be additional options around font and formatting, and we can put in additional feature requests around that. For the time being, what is the feedback around these changes?

**Comment by hifi-gustavo** [comment]
Build console output for commit: a974282bdfbf4a456c5255a3f2bd6e49835c6339 available at: https://gist.github.com/809718767189a4d799949b17da54bb9f<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9858-a974282.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9858-a974282.dmg

File: interface/src/ui/BaseLogDialog.cpp [hunk 1]
```diff
@@ -187,20 +194,36 @@ void BaseLogDialog::updateSelection() {
     }
 }
 
-KeywordHighlighter::KeywordHighlighter(QTextDocument* parent) : QSyntaxHighlighter(parent) {
+Highlighter::Highlighter(QTextDocument* parent) : QSyntaxHighlighter(parent) {
+    boldFormat.setFontWeight(75);
```
Comment (AndrewMeadows) [inline interface/src/ui/BaseLogDialog.cpp]: According to coding standard section **3.5.1.1** magic numbers like the 75 above should be put into a declared constant and given a name like: FONT_WEIGHT.

https://github.com/highfidelity/hifi/wiki/Coding-Standard#3511-the-use-of-magic-numbers-in-the-code-should-be-avoided

**Comment by kunalgosar** [comment]
Please do not merge yet.

**Comment by hifi-gustavo** [comment]
Build console output for commit: e7a16d4294a26c724b5f30dabf504088227ed7c9 available at: https://gist.github.com/809718767189a4d799949b17da54bb9f<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9858-e7a16d4.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9858-e7a16d4.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 0cee9ee172e6d864af514a386e2b72c7fba4af6d available at: https://gist.github.com/809718767189a4d799949b17da54bb9f<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9858-0cee9ee.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9858-0cee9ee.dmg

**Pull request merged**